.github/workflows/ci-cd.yml

```diff
@@ -33,6 +34,23 @@ jobs:
       - name: Unit test
         run: make test
 
+      - name: Starts the services as a dameon
+        run: make develop-daemon
+
+      - name: Run Cypress tests
```

Comment:
nice, but your pipeline is called `unit-tests`... You should maybe rename it now that it also does e2e tests.